### ISSUE TITLE
Steamworks Integration Not Working Through Proton After Recent Library Update

### ISSUE THREAD

**Issue by d10sfan** [ValveSoftware/steam-for-linux#6628]
#### Your system information

* Steam client version (build number or date): Oct 30, 20:05:34
* Distribution (e.g. Ubuntu): Manjaro
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] No

#### Please describe your issue in as much detail as possible:
Many games that were working ok before the new library update through Proton are no longer working. The common thread appears to be steamworks integration. Before the library (using the old one), these games worked fine.

For example, Space Engineers will crash with "../steamnetworkingsockets/clientlib/csteamnetworkingsockets_steamclient.cpp (261) : Assertion Failed: IClientNetworkingUtils_Create - Don't know Steam universe.  Order of init bug?"

Empyrion will say in the menu to start steam, when it's already started.

Stormworks will freeze on the menu.

#### Steps for reproducing this issue:

1. Load a game that uses steamworks through Proton
2. Except the game to work, either get freezes or events seeminly indicating that the steam integration is not working

**Comment by Evernow** [comment]
On Manjaro and I just tried Space Engineers and it works flawlessly(for this game) with no differences. Do you have only the "steam-manjaro" package installed? I only have that package, if you have the steam native package for some reason try the package provided by Manjaro

**Comment by d10sfan** [comment]
It's been working for me for at least a week now before this issue. 

I've also got the steam-manjaro package, not the steam-native one. I don't have the linux-steam-integration package installed.

**Comment by Evernow** [comment]
Did you try running: 

sudo pacman-mirrors -f 5 && sudo pacman -Syyu

And also try restarting your system?

**Comment by d10sfan** [comment]
I updated my packages from the most recent update and restarted, got the same result.

I tried that command and it said "there is nothing to do", this was after doing the package update.

I also tried renaming my ~/.steam/steam directory to force it to download everything fresh. That seemed to fix it. I'm guessing something must have happened to my local database. Things seem to be back to normal now.

Thanks for the help!
